classes/kernel.oeclass

```diff
@@ -287,12 +291,12 @@ do_compile_kernel_dtc() {
 
 KERNEL_COMPILE_POSTFUNCS:>USE_kernel_dts += " do_compile_kernel_dts"
 do_compile_kernel_dts() {
-	oe_runmake "${KERNEL_DTB_FILE}"
+	oe_runmake "${KERNEL_DTB_VENDOR}/${KERNEL_DTB_FILE}"
```

Comment:
I was afraid the this would break if `${KERNEL_DTB_VENDOR}` is empty, but my test shows otherwise:
```
% make ARCH=arm CROSS_COMPILE=arm-cortexa9neon-linux-gnueabi- /imx6dl-wandboard.dtb
  DTC     arch/arm/boot/dts//imx6dl-wandboard.dtb
```